agents/smurf_file_emulator/smurf_file_emulator.py

```diff
@@ -0,0 +1,165 @@
+import time
+import os
+import argparse
+import warnings
+import txaio
+
+from typing import Optional
+
+on_rtd = os.environ.get('READTHEDOCS') == 'True'
+if not on_rtd:
+    from ocs import ocs_agent, site_config
```

Comment:
With the newish mocking of packages in the readthedocs conf.py we no longer need this protection. And actually this protection breaks the readthedocs build. See: https://readthedocs.org/projects/socs/builds/15307869/